sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ### 4.3.0b3 (Unreleased)
 
+>[!WARNING]
+>The default `Session` consistency bugfix will impact customers whose database accounts have a `Bounded Staleness` or `Strong`
+> consistency level, and were previously not sending `Session` as a consistency_level parameter when initializing
+> their clients.
+> Default consistency level for the sync and async clients is no longer "Session" and will instead be set to the 
+  consistency level of the user's cosmos account setting on initialization if not passed during client initialization. > Please see [Consistency Levels in Azure Cosmos DB](https://docs.microsoft.com/azure/cosmos-db/consistency-levels) for more details.
```

Comment:
Maybe we could add a section in the root readme about configuring the Client consistency_level, so that we can link to that section from here in case customers need to see a code snippet to update their clients?

Reply:
agreed, and done